python/ql/lib/ext/StdLib.model.yml

```diff
@@ -12,8 +12,26 @@ extensions:
   - addsTo:
       pack: codeql/python-all
       extensible: summaryModel
-    data: []
-
+    data:
+      # See https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote
+      - ["urllib", "Member[parse].Member[quote]", "Argument[0,string:]", "ReturnValue", "taint"]
+      # See https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus
+      - ["urllib", "Member[parse].Member[quote_plus]", "Argument[0,string:]", "ReturnValue", "taint"]
+      # See https://epydoc.sourceforge.net/stdlib/urllib-module.html
+      - ["urllib", "Member[parse].Member[splitquery]", "Argument[0,url:]", "ReturnValue.TupleElement[0]", "taint"]
+      - ["urllib", "Member[parse].Member[splitquery]", "Argument[0,url:]", "ReturnValue.TupleElement[1]", "taint"]
```

Comment:
Would be nice if you could do `"ReturnValue.TupleElement[0,1]"`

Reply:
Hm, maybe we already can. I should add a test... :-)